.travis.yml

```diff
@@ -79,16 +92,10 @@ before_install:
     fi
 
   - export XMLLINT_INDENT="	"
-  - composer require squizlabs/php_codesniffer:${PHPCS_BRANCH} wp-coding-standards/wpcs:${WPCS_BRANCH} --no-update --no-suggest --no-scripts
```

Comment:
This command which has now been removed, is still needed...

Should be placed after the `minimum-stability` line and before the `composer install`.